scripts/record_deployments.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from datetime import datetime
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+# API keys and IDs are set with environment variables rather than on the
+# command line, to avoid displaying the key in logs
+NR_API_KEY_NAME = "NEW_RELIC_API_KEY"
+NR_APP_IDS_NAME = "NEW_RELIC_APP_IDS"
+SC_API_KEY_NAME = "SPEEDCURVE_API_KEY"
+SC_SIDE_ID_NAME = "SPEEDCURVE_SITE_ID"
+
+APP_NAME = os.getenv("APP_NAME")  # "wagtail" is APP_NAME -- see Makefile's constants
```

Comment:
Am not sure whether we should change "wagtail" to "developerportal" or not - does it matter massively?